analysis_options.yaml

```diff
@@ -81,6 +81,6 @@ dart_code_metrics:
   metrics:
     number-of-arguments: 8
     number-of-methods: 28
-    lines-of-executable-code: 200
+    lines-of-executable-code: 400
```

Comment:
That is too many lines IMO, I think we should think on better ways of splitting the code instead of raising this so much

Reply:
this is just because of the test file. I joined everything together because I thought it would be clearer but I can split into camera test, viewport test, etc.

Reply:
On spydon pr, the tests will be excluded from metrics, so that will solve you problem here too

Reply:
I added a comment, we should just exclude everything

Reply:
I disagree, I added a comment too.

Reply:
we can open a separated issue for this

Reply:
You probably don't even need this anymore, just rebase yourself PR with master, Lukas did excluded the tests from metrics, so your test will be fine

Reply:
yes, but I think we should treat our test code as well as treat src

Reply:
I don't think so, it is really common to have big test files, and it is not such a bad thing. But it bad to have big files on src, other metrics also falls on the same thing

Reply:
If it is so easy to wire in the game, I don't see the point of not doing, but anyway, I think we will never come to a common sense about this by that points.

I have made another comment on the PR regarding the gesture handling that I think related to this, take a look on it please.

Reply:
will just drop this change but we need a better way to handle it